src/components/graphs/RecordingGraph.svelte

```diff
@@ -213,15 +214,38 @@
 
   let canvas: HTMLCanvasElement;
   onMount(() => {
-    Chart.unregister(...registerables);
-    Chart.register([LinearScale, LineController, PointElement, LineElement]);
-    const chart = new Chart(
-      canvas.getContext('2d') ?? new HTMLCanvasElement(),
-      getConfig(),
-    );
-    return () => {
-      chart.destroy();
-    };
+    // Create the chart
+    var chart = new SmoothieChart({
+      maxValue: 2.3,
+      minValue: -2,
+      scaleSmoothing: 1,
+      millisPerPixel: 10,
+      grid: {
+        fillStyle: '#ffffff00',
+        strokeStyle: '#ffffff00',
+        millisPerLine: 3000,
+        borderVisible: false,
+      },
+      interpolation: 'linear',
+    });
+    var timeSeries = new TimeSeries();
+    chart.addTimeSeries(timeSeries, { lineWidth: 2, strokeStyle: '#00ff00' });
+
+    // Build some sample data
+    var endTime = Date.now();
+    var myvalues = [0.5, 1, 0.5, 1];
+    for (var i = 0; i < 24; i++) {
+      timeSeries.append(new Date(endTime - i * 500).getTime(), myvalues[i]);
+    }
+
+    // Render it
+    if (canvas) {
+      // Hack to set this.canvas as canvas in smoothie
+      chart.streamTo(<HTMLCanvasElement>canvas, 0);
+      chart.stop();
```

Comment:
Same issue as https://groups.google.com/g/smoothie-charts/c/VLQ4LV2EuNE/m/l5BDJEeKAwAJ, where `this.canvas` is not set properly when calling `chart.render` directly. So this is a hack for setting it without having to alter smoothie chart code